src/control/keyboard_config.cpp

```diff
@@ -16,38 +16,56 @@
 
 #include "control/keyboard_config.hpp"
 
+#include <regex>
```

Comment:
What do you need the include for?

Reply:
Code experimentation artifact. I should get myself some playground for this :laughing: